lib/torque/animator.js

```diff
@@ -1,4 +1,5 @@
 var torque = require('./');
+var StepsRange = require('./animator-steps-range');
```

Comment:
Good point, I'll rename it to `AnimatorStepsRange` to match the file/definition then